src/MultiTypeSymbolTable.chpl

```diff
@@ -66,7 +66,7 @@ module MultiTypeSymbolTable
                                    moduleName=getModuleName(),
                                    errorClass="ErrorWithContext");
             }
-            tab.remove(name); // clear out entry for name
+            //tab.remove(name); // clear out entry for name
```

Comment:
should unregister remove the `name` from the symbol table or just the `registry`